src/qml_benchmarks/models/energy_based_model.py

```diff
@@ -128,4 +131,8 @@ def score(self, X: np.ndarray, y: np.ndarray) -> float:
         if self.score_fn == 'pseudolikelihood':
             return float(np.mean(super().score_samples(X)))
         elif self.score_fn == 'mmd':
-            return float(-mmd_loss(X, self.sample(self.mmd_kwargs['n_samples']), self.mmd_kwargs['sigma']))
\ No newline at end of file
+            sigma = self.mmd_kwargs['sigma']
+            sigmas = [sigma] if isinstance(sigma, (int, float)) else sigma
+            score = np.mean([mmd_loss(X, self.sample(self.mmd_kwargs['n_samples'],
+                                                     self.mmd_kwargs['n_steps']), sigma) for sigma in sigmas])
+            return float(-score)
```

Comment:
If one gives a list of sigmas, the mean of the mmd values is returned as the score now. Behaviour is the same as before otherwise.